Flick/Networking/NetworkManager.swift

```diff
@@ -127,9 +127,26 @@ class NetworkManager {
         }
     }
 
+    /// [POST] Check if a username exists [updated as of 1/5/20]
+    static func checkUsernameExists(username: String, completion: @escaping (Bool) -> Void) {
+        let parameters: [String: Any] = [
+            "username": username
+        ]
+
+        AF.request("\(hostEndpoint)/api/username/", method: .post, parameters: parameters, encoding: JSONEncoding.default).validate().responseData { response in
+            switch response.result {
+            case .success:
+                completion(true)
```

Comment:
is this true for username exists? or true for username doesn't exist so that we can use this username to set new one?

Reply:
lol it's true if the username doesn't exists. I'll make this clearer, sorry my naming is kinda confusing